agent/agent_test.go

```diff
@@ -10,7 +10,6 @@ import (
 	"github.com/google/go-tpm/tpm2"
 	"github.com/google/go-tpm/tpm2/transport"
 	"github.com/google/go-tpm/tpm2/transport/simulator"
-	"golang.org/x/crypto/ssh/agent"
```

Comment:
Otherwise https://github.com/Foxboron/ssh-tpm-agent/actions/runs/9924219085/job/27415292916?pr=57

Reply:
Maybe I don't understang `go`, but something weird is happening in the CI.
`Make test` passes locally.